models/article-models.js

```diff
@@ -51,3 +51,28 @@ exports.postNewComment = (article_id, username, body) => {
       return rows[0];
     });
 };
+
+exports.patchArticleById = (article_id, inc_votes) => {
+  return db
+    .query(`SELECT * FROM articles WHERE articles.article_id = $1`, [
```

Comment:
Rather than doing another query here to check if the ID exists, you may wish to reuse your earlier model to get article  by id. This could be used in your controller to check this.